### PR TITLE
Fixes engraved messages

### PR DESCRIPTION
**Pull request by coiax**
:cl: coiax
fix: Curator soapstones now successfully leave messages for future
shifts.
del: Soapstones can no longer be purchased in cargo.
del: The janitor no longer starts with an empty soapstone.
experiment: Engraved messages can be left anywhere in the world, but be
wary that the terrain of places like lavaland and space can change shift
to shift.
/:cl:

- Curators are now the sole owners of soapstones at shift start. This
means that a huge cargo point bounty shift will not overwhelm the
station with messages.
- Curators have an additional thing to curate.
- All the snowflake name bullshit is dumb, they're called soapstones,
they're plasma bluespace chisels, okay

### CODE REVIEW

**Comment by TehZombehz** [comment]
This was being abused again, wasn't it?

File: code/modules/library/soapstone.dm [hunk 1]
```diff
@@ -181,6 +151,8 @@
 	var/hash = md5(hidden_message)
 	var/newcolor = copytext(hash, 1, 7)
 	add_atom_colour("#[newcolor]", FIXED_COLOUR_PRIORITY)
+	light_color = "#[newcolor]"
+	set_light(1)
```
Comment (ChangelingRain) [inline code/modules/library/soapstone.dm]: should be something like set_light(1.4, 0.5) as 1.4 is the minimum range.

**Comment by coiax** [comment]
@TehZombehz actually I just broke them saving from round to round

File: code/modules/library/soapstone.dm (hunk 1 above)
Comment (coiax) [inline code/modules/library/soapstone.dm]: Eh, I like the current amount of light they give out. You can only really notice it in the dark.

File: code/controllers/subsystem/persistence.dm [hunk 2]
```diff
@@ -104,10 +103,10 @@ SUBSYSTEM_DEF(persistence)
...
 		var/obj/structure/chisel_message/M = new(T)
 
-		M.unpack(item)
-		if(!M.loc)
-			M.persists = FALSE
-			qdel(M)
+		if(!QDELETED(M))
+			M.unpack(item)
+
+	world.log << "Loaded [saved_messages.len] engraved messages on map [SSmapping.config.map_name]"
```
Comment (lzimann) [inline code/controllers/subsystem/persistence.dm]: `log_world` please

File: code/controllers/subsystem/persistence.dm [hunk 3]
```diff
@@ -175,9 +174,16 @@ SUBSYSTEM_DEF(persistence)
...
 /datum/controller/subsystem/persistence/proc/CollectChiselMessages()
+	var/savefile/chisel_messages_sav = new /savefile("data/npc_saves/ChiselMessages.sav")
+
+	// For debugging purposes.
+	chisel_messages_sav.ExportText("/", "data/npc_saves/ChiselMessages.txt")
+
 	for(var/obj/structure/chisel_message/M in chisel_messages)
 		saved_messages += list(M.pack())
 
+	world.log << "Saved [saved_messages.len] engraved messages on map [SSmapping.config.map_name]"
```
Comment (lzimann) [inline code/controllers/subsystem/persistence.dm]: `log_world`

File: code/controllers/subsystem/persistence.dm [hunk 4]
```diff
@@ -175,9 +174,16 @@ SUBSYSTEM_DEF(persistence)
 	secret_satchels[SSmapping.config.map_name] << old_secret_satchels
 
 /datum/controller/subsystem/persistence/proc/CollectChiselMessages()
+	var/savefile/chisel_messages_sav = new /savefile("data/npc_saves/ChiselMessages.sav")
+
+	// For debugging purposes.
+	chisel_messages_sav.ExportText("/", "data/npc_saves/ChiselMessages.txt")
```
Comment (AnturK) [inline code/controllers/subsystem/persistence.dm]: Remove this and this is good to go

**Comment by coiax** [comment]
Done.

**Pull request merged**